src/frontend/src/optimizer/plan_node/stream_table_scan.rs

```diff
@@ -92,9 +92,15 @@ impl StreamTableScan {
         use risingwave_pb::stream_plan::*;
 
         let batch_plan_node = BatchPlanNode {
-            table_ref_id: Some(TableRefId {
-                table_id: self.logical.table_desc().table_id.table_id as i32,
-                schema_ref_id: Default::default(),
+            table_desc: Some(CellBasedTableDesc {
+                table_id: self.logical.table_desc().table_id.into(),
+                pk: self
```

Comment:
The field name should be order_key instead? pk is different from order key in the streaming engine.

Reply:
Ineed. Maybe Meta should embed this info into Stream Plan, so executor can just read it and build state table.

Reply:
Updated: https://github.com/singularity-data/risingwave/pull/3060/commits/2206ad566848b65f8b9d1411802e38d5cfe40133

Reply:
updated: https://github.com/singularity-data/risingwave/pull/3060/files/b4635acce4dbe7f03e41c6031d819e88acf5ffc1..598aadb3f6a233046c50c1ccfdcc0324a3fd4499